hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -143,6 +143,8 @@
         },
       ]
     },
+
+// TODO(#1412):
```

Comment:
I just came across this and added a note, since I will be using the same mechanism for this in the pinmux.